src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -405,6 +405,50 @@ async Task IDurableOrchestrationClient.TerminateAsync(string instanceId, string
             }
         }
 
+        async Task IDurableOrchestrationClient.SuspendAsync(string instanceId, string reason)
+        {
+            OrchestrationState state = await this.GetOrchestrationInstanceStateAsync(instanceId);
+            if (IsOrchestrationSuspendable(state))
+            {
+                state.OrchestrationInstance.ExecutionId = null;
+
+                await this.client.SuspendInstanceAsync(state.OrchestrationInstance, reason);
+
+                this.traceHelper.SuspendingOrchestration(this.TaskHubName, state.Name, instanceId, reason);
```

Comment:
Let's move this trace above the call to `SuspendInstanceAsync`.

Reply:
Sounds good.